cmd/oras/internal/display/track/target.go

```diff
@@ -112,3 +113,13 @@ func (t *graphTarget) Prompt(desc ocispec.Descriptor, prompt string) error {
 	status <- progress.EndTiming()
 	return nil
 }
+
+// Inner returns the inner oras.GraphTarget.
+func (t *graphTarget) Inner() oras.GraphTarget {
+	return t.GraphTarget
+}
+
+// Inner returns the inner oras.GraphTarget.
+func (t *referenceGraphTarget) Inner() oras.GraphTarget {
+	return t.GraphTarget
+}
```

Comment:
This looks like redundant as `*referenceGraphTarget` already has `Inner()` from `*graphTarget`.